estate/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Real Estate",
+    "description": "Real Estate Advertising",
+    "author": "robal",
```

Comment:
```suggestion
    "author": "Odoo S.A.",
```
As an employee, you work in the name of Odoo.